x-pack/plugins/observability_solution/observability/server/plugin.ts

```diff
@@ -106,112 +107,8 @@ export class ObservabilityPlugin implements Plugin<ObservabilityPluginSetup> {
 
     const alertDetailsContextualInsightsService = new AlertDetailsContextualInsightsService();
 
-    plugins.features.registerKibanaFeature({
```

Comment:
Moved to `cases_v1` here: https://github.com/elastic/kibana/pull/194898/files#diff-55346195e13917d2cb66f96dac9547706b4bc55e0765f77c026bd5aafcf04ec7